src/Domain.h

```diff
@@ -473,88 +446,12 @@ class Domain {
 	unsigned _globalUSteps;
 	double _globalSigmaU;
 	double _globalSigmaUU;
-
-	//! 1 / dimension of a profile cuboid
-	double _universalInvProfileUnit[3];
-	//! number of successive profile cuboids in x/y/z direction
-	unsigned _universalNProfileUnits[3];
-	//! local N profile map
-	std::map<unsigned, long double> _localNProfile;
-	//! global N profile map
-	std::map<unsigned, double> _universalNProfile;
-	//! local directed velocity profile map
-	std::map<unsigned, long double> _localvProfile[3];
-	//! global directed velocity  profile map
-	std::map<unsigned, double> _universalvProfile[3];
-	//! local kinetic energy profile map
-	std::map<unsigned, long double> _localKineticProfile;
-	//! global kinetic energy profile map
-	std::map<unsigned, double> _universalKineticProfile;
-	//! local counter w. r. t. degrees of freedom
-	std::map<unsigned, long double> _localDOFProfile;
-	//! global counter w. r. t. degrees of freedom
-	std::map<unsigned, double> _universalDOFProfile;
-	//! how many _evaluated_ timesteps are currently accumulated in the profile?
-	unsigned _globalAccumulatedDatasets;
 	//! which components should be considered?
 	std::map<unsigned, bool> _universalProfiledComponents;
         double _universalProfiledComponentMass;  // set from outside
-	// //! local virial / pressure profile map
-	// std::map<unsigned, long double> _localPDProfile;
-	// //! global virial / pressure profile map
-	// std::map<unsigned, double> _universalPDProfile;
-	//! local virial / pressure profile map
-	std::map<unsigned, long double> _localPXProfile;
-	//! global virial / pressure profile map
-	std::map<unsigned, double> _universalPXProfile;
-	//! local virial / pressure profile map
-	std::map<unsigned, long double> _localPYProfile;
-	//! global virial / pressure profile map
-	std::map<unsigned, double> _universalPYProfile;
-	//! local virial / pressure profile map
-	std::map<unsigned, long double> _localPZProfile;
-	//! global virial / pressure profile map
-	std::map<unsigned, double> _universalPZProfile;
-
-        std::map<unsigned, double> _universalTProfile; 
-        std::map<unsigned, long double> _localWidomProfile;  // submit individually
```

Comment:
What was the case with these Widom and Virial Profiles again? 

I'd say to merge now, to get these changes in, but maybe think about restoring them later, cleanly.

Reply:
As far as I could tell, they were not fully implemented or were not written out properly.
DOF and Kinetic Profiles are already implemented in the new plugin version.
Missing functionality could be easily added via an extension of `ProfileBase`, but it was unclear to me what they were supposed to do from the code given in `Domain.cpp`